tests/css-pseudo-4.js

```diff
@@ -56,6 +56,15 @@ export default {
 			},
 			tests: ['::grammar-error'],
 		},
+		// This is also defined in css-highlight-api-1
```

Comment:
Re-reading the spec., I actually think it makes more sense to _not_ add this test here as we have it already for css-highlight-api-1. And we generally only test features in the specs they were introduced in.

Sorry for the confusion!

Reply:
So let's just add `::details-content::highlight()` ?

Reply:
Yes. Otherwise we would just duplicate the test of css-highlight-api-1.

And again, sorry for not realizing that earlier! Otherwise we could have put that in the other PR.

Reply:
It doesn't matter.

All done